### PR TITLE
Added textColor for android

### LINKED ISSUE #6: Snackbar text black on older devices
The text for the Snackbar appears black on devices using android 4.4. 

screen shot 2016-12-08 at 9 18 45 pm

Currently making this change to SnackbarModule.java to display it correctly.
```java
Snackbar snackbar = Snackbar.make(view, Html.fromHtml("<font color=\"#ffffff\">" + title + "</font>"), duration);

```

### PR DESCRIPTION
**Pull request by iRoachie**
Adds tintColor param for android. Currently only uses hex values. By default tintColor is white if no tintColor is explicitly set (Fixes #6)

### CODE REVIEW

**Comment by cooperka** [comment]
Thanks @iRoachie! Code logic LGTM, I'll test it out soon before merging.

File: index.js [hunk 1]
```diff
@@ -12,6 +12,11 @@ export default {
     if (options.action && options.action.color) {
       options.action.color = processColor(options.action.color);
     }
+
+     if(!options.tintColor) {
```
Comment (cooperka) [inline index.js]: Could we call this `textColor` instead? And it should probably also be added to the docs.

**Review by cooperka — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (iRoachie) [reply index.js]: Oh I just called it tintColor cause that's what it's called on all the other react native components.

Comment (cooperka) [reply index.js]: I'm not sure which components you're referring to, but I think generally tint is used for images (not text). Android also uses textColor to specify this on their [native snackbars](https://developer.android.com/reference/android/support/design/widget/Snackbar.html).

Comment (iRoachie) [reply index.js]: Alright will make the change

**Review by iRoachie — COMMENTED** [review]
Changes added. Also documented

**Comment by cooperka** [comment]
@iRoachie I tested it out and it turns out there's a [better fix](https://github.com/cooperka/react-native-snackbar/commit/85b089045f4647876732680837b02c0e062edd9f) that I've implemented already. This PR is still valid though -- I think it would be great to be able to set textColor, unrelated to #6. I'll rebase this on the latest master soon and merge. Thanks again for your help!

**Comment by iRoachie** [comment]
Alright then great! I'll cancel this one and come up with another way of setting textColor using the method you used. Thanks

**Comment by cooperka** [comment]
You could even keep this PR open if you prefer, since the title is still accurate and the code is pretty much the same! I was thinking instead of hardcoding `snackbarText.setTextColor(Color.WHITE);` we could just pass in the color that the user specified. Should hopefully be pretty straightforward.

**Comment by iRoachie** [comment]
We would need someway to map colours (rgb, solids, hex) to the respect android IOS Color classes

**Pull request closed without merging**